docs/modules/operation/pages/deep-dive/alarms/configuring-alarms.adoc

```diff
@@ -48,18 +46,21 @@ NOTE: Any Event that is marked as `donotpersist` in the logmsg element's `dest`
   </restriction>
 </simpleType>
 ----
-NOTE: See also: <<<deep-dive/events/event-definition.adoc#ga-events-anatomy-of-an-event, anatomy of an event>>
 
-== The reduction-key
+See <<deep-dive/events/event-definition.adoc#ga-events-anatomy-of-an-event, Anatomy of an event>> for more information.
+
+== Attributes and elements
+
+=== reduction-key
 
-Alarmd is designed to reduce multiple occurrences of an alarm into a single alarm.
-The critical attribute when defining the alarm-data of an event is the `reduction-key`.
-This attribute can contain literal strings as well as references to properties (fields and parameters) of the event.
-The reduction-key uniquely identifies the signature of a problem and, as such, is used to reduce (de-duplicate) events so that only one problem is instantiated.
-Most commonly, the event's identifier (UEI) is used as the left-most (least significant) portion of the reduction-key, followed by other properties of the event from least to most significant and, traditionally, separated with the literal `:`.
+Alarmd is designed to consolidate multiple occurrences of an alarm into a single alarm.
+When defining an event's `alarm-data`, the critical attribute is the `reduction-key`.
+It can contain literal strings as well as references to the event's properties (fields and parameters).
 
-.Multi-part reduction-key
-====
+The `reduction-key` uniquely identifies a problem's signature and, as such, is used to reduce events so that only one problem is instantiated.
```

Comment:
```suggestion
The `reduction-key` uniquely identifies a problem's signature and is used to reduce events so that only one problem is instantiated.
```